tools/lkl/tests/hijack-test.sh

```diff
@@ -43,10 +45,14 @@ ${hijack_script} ./ping6 -c 1 ::1
 rm ./ping6
 
 echo "== Mount/dump tests =="
+cfgjson=${work_dir}/hijack-test.conf
+echo " {" \
```

Comment:
Lets use <<EOF style indirection to avoid escaping " and make it more readable.

Reply:
Thank you for your review.
I have fixed.